pkg/cloudprovider/aws/block_store.go

```diff
@@ -180,6 +184,29 @@ func (b *blockStore) CreateSnapshot(volumeID, volumeAZ string, tags map[string]s
 	return *res.SnapshotId, nil
 }
 
+func getTagsForCluster(snapshotTags []*ec2.Tag) []*ec2.Tag {
+	var result []*ec2.Tag
+
+	if clusterName, found := os.LookupEnv("AWS_CLUSTER_NAME"); found {
+		result = append(result, ec2Tag("kubernetes.io/cluster/"+clusterName, "owned"))
+		result = append(result, ec2Tag("KubernetesCluster", clusterName))
+	}
+
+	_, haveAWSClusterNameEnvVar := os.LookupEnv("AWS_CLUSTER_NAME")
```

Comment:
I wouldn't look this up a second time, since you added it above. I'd rename `found` above to `haveAWSClusterNameEnvVar` and remove this line.

Reply:
I moved the lookup above the first if statement and do it only once, then only use the `haveAWSClusterNameEnvVar`, is that ok?